spec/system/admin/products_v3/products_spec.rb

```diff
@@ -2,13 +2,16 @@
 
 require "system_helper"
 
-describe 'As an admin, I can manage products', feature: :admin_style_v3 do
+describe 'As an enterprise user, I can manage my products', feature: :admin_style_v3 do
   include WebHelper
   include AuthenticationHelper
   include FileHelper
 
+  let(:producer) { create(:supplier_enterprise) }
```

Comment:
In many places the codebase says "supplier". We now prefer the term "producer", which is used in general communication, so I've named the variables as such. 
Maybe one day we'll update the codebase to avoid confusion..

Reply:
Oh, I didn't actually know this. "Supplier" is a bit more general. The enterprise may not be producing the product. :shrug:

Reply:
I recall having a conversation in Slack I think, and that "producer" was the winner.
I agree that "supplier" makes more sense, although arguably the distributor also "supplies" the product, so it could be ambiguous...

I can't find that conversation though, so now I'm not sure! I will seek clarification in #product-circle.

Reply:
I think this is the conversation https://openfoodnetwork.slack.com/archives/C01T75H6G0Z/p1679368367236359
Seems we ended up with "supplier" back then 🙂

Reply:
Thanks Sigmund, I forgot my own comments! 😅

I'm going to write it on a post-it note on my computer screen..